hack/make-rules/test-e2e-node.sh

```diff
@@ -73,9 +73,9 @@ if [ $remote = true ] ; then
   gubernator=${GUBERNATOR:-"false"}
   image_config_file=${IMAGE_CONFIG_FILE:-""}
   if [[ $hosts == "" && $images == "" && $image_config_file == "" ]]; then
-    image_project=${IMAGE_PROJECT:-"google-containers"}
+    image_project=${IMAGE_PROJECT:-"cos-cloud"}
     gci_image=$(gcloud compute images list --project $image_project \
-    --no-standard-images --regexp="gci-dev.*" --format="table[no-heading](name)")
+    --no-standard-images --regexp="cos-dev.*" --format="table[no-heading](name)")
```

Comment:
`dev` was a thing of the past. The idea was to test `dev` images as soon as they are released. Now, we can switch to `beta` I hope. We aren't updating gci images as frequently as we used to do.